csharp-advanced-topics/DeprecationInCsharp/DeprecationInCsharp/StringUtils.cs

```diff
@@ -1,11 +1,9 @@
 ﻿using System.Globalization;
 
-namespace Deprecation
+namespace Deprecation.StringUtils
```

Comment:
A file scoped namespace allows us to reduce nesting, so instead of having:
`namespace MyNamespace`
`{`
`    // some code`
`}`
we can write:
`namespace MyNamespace;`
`// some code`
No brackets needed and so we are able to reduce the nesting withing the file which aids in readability. You can read more here [File scoped namespace](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces). Sorry if I didn't link the documentation earlier, I had intended to.

Reply:
Thanks for detailed answer i have implemented same you check